activation/post_supervisor.go

```diff
@@ -46,12 +47,18 @@ func MainnetPostServiceConfig() PostSupervisorConfig {
 type PostSupervisorConfig struct {
 	PostServiceCmd string `mapstructure:"post-opts-post-service"`
 
-	DataDir         string        `mapstructure:"post-opts-datadir"`
-	NodeAddress     string        `mapstructure:"post-opts-node-address"`
-	PowDifficulty   PowDifficulty `mapstructure:"post-opts-pow-difficulty"`
-	PostServiceMode string        `mapstructure:"post-opts-post-service-mode"`
+	DataDir         string `mapstructure:"post-opts-datadir"`
+	NodeAddress     string `mapstructure:"post-opts-node-address"`
+	PostServiceMode string `mapstructure:"post-opts-post-service-mode"`
 
-	N string `mapstructure:"post-opts-n"`
+	PowDifficulty PowDifficulty `mapstructure:"post-opts-pow-difficulty"`
+	K1            uint32        `mapstructure:"post-opts-k1"`
+	K2            uint32        `mapstructure:"post-opts-k2"`
+	K3            uint32        `mapstructure:"post-opts-k3"`
+
+	N uint `mapstructure:"post-opts-n"`
+	R uint `mapstructure:"post-opts-r"`
+	P uint `mapstructure:"post-opts-p"`
```

Comment:
Why duplicate the POST network configuration? The supervisor could reuse the POST config we already have defined.

Reply:
Primarily for easier use in tests, I'll look into de-duplicating configuration where possible.

Reply:
Deduplicated parameters where possible.